src/main/resources/l10n/JabRef_en.properties

```diff
@@ -271,6 +271,7 @@ Each\ line\ must\ be\ of\ the\ following\ form\:\ \'tab\:field1;field2;...;field
 
 Search\ groups\ migration\ of\ %0=Search groups migration of %0
 The\ search\ groups\ syntax\ is\ outdated.\ Do\ you\ want\ to\ migrate\ to\ the\ new\ syntax?=The search groups syntax is outdated. Do you want to migrate to the new syntax?
+The\ search\ groups\ syntax\ has\ been\ reverted\ to\ the\ old\ one.\ Please\ use\ the\ backup\ you\ made\ before\ migrating\ to\ 6.0-alpha. = The search groups syntax has been reverted to the old one. Please use the backup you made before migrating to 6.0-alpha.
```

Comment:
No need for translation. We will remove this after 6.0 release. It is OK to have it untranslated in JabRef.